src/TorchSharp/Tensor/torch.ComparisonOps.cs

```diff
@@ -276,6 +276,21 @@ public static Tensor searchsorted(Tensor sorted_sequence, Scalar values, bool ou
             return new Tensor(res);
         }
 
+        /// https://github.com/numpy/numpy/blob/v1.24.0/numpy/lib/histograms.py#L679
```

Comment:
I know this is a little bit of a nit-pick, but does this method belong in a file called 'ComparisonOps'?